src/ImageSharp/MetaData/ImageMetaData.cs

```diff
@@ -0,0 +1,169 @@
+            {
+              this.Properties.Add(new ImageProperty(property));
+            }
+
+            if (other.ExifProfile != null)
+            {
+              this.ExifProfile = new ExifProfile(other.ExifProfile);
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the resolution of the image in x- direction. It is defined as
+        ///  number of dots per inch and should be an positive value.
+        /// </summary>
+        /// <value>The density of the image in x- direction.</value>
+        public double HorizontalResolution
+        {
+            get
+            {
+                return this.horizontalResolution;
+            }
+
+            set
+            {
+              if (value > 0)
+              {
+                  this.horizontalResolution = value;
+              }
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the resolution of the image in y- direction. It is defined as
+        /// number of dots per inch and should be an positive value.
+        /// </summary>
+        /// <value>The density of the image in y- direction.</value>
+        public double VerticalResolution
+        {
+            get
+            {
+                return this.verticalResolution;
+            }
+
+            set
+            {
+                if (value > 0)
+                {
+                    this.verticalResolution = value;
+                }
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the Exif profile.
+        /// </summary>
+        public ExifProfile ExifProfile { get; set; }
+
+        /// <summary>
+        /// Gets or sets the frame delay for animated images.
+        /// If not 0, this field specifies the number of hundredths (1/100) of a second to
+        /// wait before continuing with the processing of the Data Stream.
+        /// The clock starts ticking immediately after the graphic is rendered.
+        /// </summary>
+        public int FrameDelay { get; set; }
+
+        /// <summary>
+        /// Gets the list of properties for storing meta information about this image.
+        /// </summary>
+        /// <value>A list of image properties.</value>
+        public IList<ImageProperty> Properties { get; } = new List<ImageProperty>();
+
+        /// <summary>
+        /// Gets or sets the quality of the image. This affects the output quality of lossy image formats.
+        /// </summary>
+        public int Quality { get; set; }
+
+        /// <summary>
+        /// Gets or sets the number of times any animation is repeated.
+        /// <remarks>0 means to repeat indefinitely.</remarks>
+        /// </summary>
+        public ushort RepeatCount { get; set; }
+
+        /// <summary>
+        /// Synchronizes the profiles with the current meta data.
+        /// </summary>
+        internal void SyncProfiles()
+        {
+            this.SyncExifProfile();
+        }
+
+        private void SyncExifProfile()
```

Comment:
Wouldn't it be better to move this method into `ExifProfile` and use it as `this.?ExifProfile.Sync(this)`?